docs/configuration.adoc

```diff
@@ -68,9 +68,44 @@ TIP: `git.commit`, `git.previous.commit` and `git.last.commits` Java system prop
 When this strategy gets all changes then inspect all tests of current project checking which ones _imports_ these classes.
 If the test exercises a business class that has been modified, we treat it as *important*  so it will be executed earlier in the test plan.
 
+.About transitivity
+****
+Currently any we are resolving in transitive way imports.
+This means that any import of a business class is considered as a dependency of the test too.
+Let's look an example:
+
+Suppose we have `ATest.java` which imports `A.java`.
+At the same time `A.java` imports `B.java` (ATest -> A -> B).
+If `B.java` is modified, then `ATest.java` is considered an *important* test too.
+
+By default this import transitivity is applied to all imports except the ones from `java` and `javax`.
+
+Sometimes you might want to stop this transitivity before reaching these imports, for example in case of developing an application with Spring, you'd probably want to *exclude* `org.springframework` imports.
+Or maybe just *include* imports from your business code, for example all imports from `org.superbiz`.
```

Comment:
I would avoid any particular library. Let's make sth up. The point is that we use additional libraries which won't change - so we want to optimize it.